src/rdo.rs

```diff
@@ -297,6 +301,7 @@ pub fn rdo_mode_decision(
   }
 
   cw.bc.set_mode(bo, bsize, best_mode_luma);
+  cw.bc.set_motion_vector(bo, bsize, best_mv);
```

Comment:
Also, unless best_mv is updated, we can do this right after "let mv = ...." ?

Reply:
No. Basically this is needed so that the RDO for the next block behaves correctly.